modules/checking-cluster-resources-after-initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,223 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="checking-cluster-resources-after-initiating-limited-live-migration_{context}"]
+= Checking cluster resources after initiating the limited live migration
+
+The following procedure shows you how to check for egress IP resources, egress firewall resources, multicast enabled namespaces, and network policies when your deploying is using OVN-Kubernetes. If you had these resources on OpenShift SDN, you should check them after migration to ensure that they are working properly.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have successfully migrated from OpenShift SDN to OVN-Kubernetes by using the limited live migration.
+
+.Procedure
+
+. As an {product-title} cluster administrator, check for egress firewall resources. You can do this by using the `oc` CLI, or by using the {product-title} web console.
+
+.. To check for egress firewall resource by using the `oc` CLI tool:
```

Comment:
This and line 65 could be unordered, no? Since you choose either one (or both)?
```suggestion
* To check for egress firewall resource by using the `oc` CLI tool:
```